RokredUI/Services/INavigationService.cs

```diff
@@ -6,7 +6,7 @@ namespace RokredUI.Services
 {
     public interface INavigationService
     {
-        IObservable<Unit> Push<T>(T viewModel) where T : class, IViewModel;
+        IObservable<Unit> Push<T>(object parameter) where T : class, IViewModel;

```

Comment:
Could be T2 and IParameter or whatever. But do you see what I mean about the parameter Edgar?

Reply:
How are we going to the parameter when we get to the new ViewModel.

Reply:
We will need to have some method that will intercept the life cycle and get the values ?

Reply:
This is a working example ;) 
Check the ViewModel1 going to ViewModel2 on my stuff I checked in

Reply:
I would like to talk to you tomorrow about this, I find out a better way